sql/information_schema/information_schema.go

```diff
@@ -1978,77 +2022,107 @@ func tablesRowIter(ctx *Context, cat Catalog) (RowIter, error) {
 			return nil, err
 		}
 
-		views, err := viewsInDatabase(ctx, db)
+		views, err := viewsInDatabase(ctx, db.database)
 		if err != nil {
 			return nil, err
 		}
 
 		for _, view := range views {
 			rows = append(rows, Row{
-				catalogName, // table_catalog
-				schemaName,  // table_schema
-				view.Name,   // table_name
-				"VIEW",      // table_type
-				nil,         // engine
-				nil,         // version (protocol, always 10)
-				nil,         // row_format
-				nil,         // table_rows
-				nil,         // avg_row_length
-				nil,         // data_length
-				nil,         // max_data_length
-				nil,         // max_data_length
-				nil,         // data_free
-				nil,         // auto_increment
-				y2k,         // create_time
-				nil,         // update_time
-				nil,         // check_time
-				nil,         // table_collation
-				nil,         // checksum
-				nil,         // create_options
-				"VIEW",      // table_comment
+				db.catalogName, // table_catalog
+				db.schemaName,  // table_schema
+				view.Name,      // table_name
+				"VIEW",         // table_type
+				nil,            // engine
+				nil,            // version (protocol, always 10)
+				nil,            // row_format
+				nil,            // table_rows
+				nil,            // avg_row_length
+				nil,            // data_length
+				nil,            // max_data_length
+				nil,            // max_data_length
+				nil,            // data_free
+				nil,            // auto_increment
+				y2k,            // create_time
+				nil,            // update_time
+				nil,            // check_time
+				nil,            // table_collation
+				nil,            // checksum
+				nil,            // create_options
+				"VIEW",         // table_comment
 			})
 		}
 	}
 
 	return RowsToRowIter(rows...), nil
 }
 
-// allDatabases expands all databases in the catalog to include all schemas if present
-func allDatabases(ctx *Context, cat Catalog) ([]Database, error) {
-	var dbs []Database
+// dbWithNames includes the Database with the catalog and schema names.
+type dbWithNames struct {
+	database    Database
+	catalogName string
+	schemaName  string
+}
+
+// allDatabases expands all databases in the catalog to include all schemas in the current database if present
```

Comment:
So the behavior is: only schemas in the current database are present in the information_schema tables?

Reply:
Might want to document that this behavior is to support postgres, we'll probably want to move that code there at some point.